packages/api2/src/quantifications/services/quantifications.service.ts

```diff
@@ -205,29 +205,22 @@ export class QuantificationsService {
       praise._id,
     );
 
-    // Filter out dismissed quantifications and quantifications that are not completed
-    const completedQuantifications = quantifications.filter((q) => {
-      if (!this.isQuantificationCompleted(q)) return false;
-      if (q.dismissed) return false;
-      return true;
-    });
-
-    // If no quantifications are completed the score is 0
-    if (completedQuantifications.length === 0) return 0;
+    if (quantifications.length === 0) return 0;
 
     // Calculate the score for each quantification
     const scores = await Promise.all(
-      completedQuantifications.map((q) => {
+      quantifications.map((q) => {
```

Comment:
Rule is still - If a quantifiacation has the score 0 it means it is not completed and should not be included in the average. On the other hand, if q quant is marked as dismissed, it is counted as zero and included in the average.